lib/src/geojson.dart

```diff
@@ -317,12 +321,28 @@ class BBox extends CoordinateType {
   bool get _is3D => length == 6;
 
   num get lng1 => _items[0];
+
   num get lat1 => _items[1];
+
   num? get alt1 => _is3D ? _items[2] : null;
+
   num get lng2 => _items[_is3D ? 3 : 2];
+
   num get lat2 => _items[_is3D ? 4 : 3];
+
   num? get alt2 => _is3D ? _items[5] : null;
 
+  BBox copyWith({
+    num? lng1,
+    num? lat1,
+    num? lng2,
+    num? lat2,
+    num? alt1,
+    num? alt2,
+  }) =>
```

Comment:
Wait a second:
The original /& default constructor of the BBox has the following signature:

```dart
BBox(
    num lng1,
    num lat1,
    num alt1,
    num lng2, [
    num? lat2,
    num? alt2,
  ]) : super([
          lng1,
          lat1,
          alt1,
          lng2,
          if (lat2 != null) lat2,
          if (alt2 != null) alt2,
        ]);
```

Reply:
We should follow the same sequence, otherwise we'll have problems

Reply:
I changed the order of the optional params(Altitudes) according to standard and moved on to change the tests. Please check the result.